demo/admin/src/products/future/ManufacturersPage.cometGen.ts

```diff
@@ -0,0 +1,16 @@
+import { type CrudPageConfig } from "@comet/admin-generator";
+import { type GQLManufacturer } from "@src/graphql.generated";
+
+import { ManufacturerForm } from "../ManufacturerForm";
+import { ManufacturersGrid } from "./generated/ManufacturersGrid";
+
+export const ManufacturersPage: CrudPageConfig<GQLManufacturer> = {
+    type: "crudPage",
```

Comment:
Why crudPage, and not page?

Reply:
I changed it to `page`: c278bddd978e15ddf6b45b84d73598882f7dea21